src/gatling/simulations/scenarios/ET_MakeAClaimPt2.scala

```diff
@@ -338,8 +338,8 @@ object ET_MakeAClaimPt2 {
         .post(BaseURL + "/respondent/1/acas-cert-num")
         .headers(CommonHeader)
         .header("content-type", "application/x-www-form-urlencoded")
-        .formParam("_csrf", "${csrf}")
-        .formParam("et-sya-session", "${etSession}")
+        .formParam("_csrf", "#{csrf}")
+        .formParam("et-sya-session", "#{etSession}")
         .formParam("acasCert", "Yes")
         .formParam("acasCertNum", "R" + Common.randomNumber(6) + "/" + Common.randomNumber(2) + "/" + Common.randomNumber(2))
```

Comment:
Same with this, every user I think will get assigned the same random numbers as they are determined once at compile time. You'll need to set them as Gatling session variables

Reply:
I have done the same as above and generated it at the start of the script instead